Bottom: 830505cbfc038c084e4e7657adeed13f78dcb97d
Top:    2e0e6fdce97ed69b7980f592ada09595ecfd2a58
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-01-03 23:43:02 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
